<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:rgb(30, 30, 30)

}
        </string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1361</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTabWidget{
background-color: rgb(90, 90, 90)
}
QTabWidget QTabBar{
background-color: rgb(98, 98, 98);
width: 220px;
font: 10pt &quot;Arial&quot;;
}


QTabWidget QTabBar::tab{
background-color: rgb(126, 126, 126);
height: 50px;
width: 220px;
color: rgb(255, 255, 255);
border-bottom:7px solid rgb(213, 71, 106);
}

QTabWidget QTabBar::tab:selected{
background-color: rgb(76, 76, 76);
width: 220px;
border-bottom:7px solid rgb(0, 202, 148)
}
QTabWidget QTabBar::tab:hover{
background-color: rgb(61, 177, 6);
width: 220px;
border-bottom:7px solid rgb(0, 202, 148)
}

QTabBar::tab:last {
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    margin-right: 600px;
}
QTabWidget&gt;QWidget&gt;QWidget{background: rgb(70, 70, 70);
background-image: url(1441804.jpg);
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Denoising</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color:rgb(104, 104, 104);
opacity:0.3;
color:white;
}</string>
     </property>
     <property name="title">
      <string>load image</string>
     </property>
     <widget class="QLineEdit" name="filename">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:rgb(255, 255, 255)
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browse">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>b</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>411</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color:rgb(104, 104, 104);
color:white;
}</string>
     </property>
     <property name="title">
      <string>noise standard diviation:</string>
     </property>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 1px solid white;
    height: 10px;
    margin: 0px;
    }
QSlider::handle:horizontal {
    background-color: white;
    border: 1px solid;
    height: 10px;
    width: 10px;
    margin: -15px 0px;
    }</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addNoise">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color:rgb(104, 104, 104);
color:white;
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>35</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color:rgb(104, 104, 104);
color:white;
}</string>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>10</y>
       <width>381</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color:rgb(104, 104, 104);

color:white;
}</string>
     </property>
     <property name="title">
      <string>Denoise method</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton{ color: rgb(255, 255, 255); background-color: rgba(255, 255, 255, 0);}</string>
      </property>
      <property name="text">
       <string>VisuShrink</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton{ color: rgb(255, 255, 255); background-color: rgba(255, 255, 255, 0);}</string>
      </property>
      <property name="text">
       <string>BayesShrink</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton{ color: rgb(255, 255, 255); background-color: rgba(255, 255, 255, 0);}</string>
      </property>
      <property name="text">
       <string>Proposed</string>
      </property>
     </widget>
     <widget class="QPushButton" name="debruit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
      </property>
      <property name="text">
       <string>debruitage</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>300</y>
       <width>431</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color:rgb(104, 104, 104);

color:white;
}</string>
     </property>
     <property name="title">
      <string>Noisy image</string>
     </property>
     <widget class="QLabel" name="bruite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>411</width>
        <height>301</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:3px dashed white;
background-color:rgb(104, 104, 104);
            }</string>
      </property>
      <property name="text">
       <string>Noisy</string>
      </property>
     </widget>
     <widget class="QLabel" name="psnrNoisy">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color:rgb(134, 134, 134);
color:white;
padding:1px
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>360</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
background-color:rgb(104, 104, 104);
            }</string>
      </property>
      <property name="text">
       <string>PSNR:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>431</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color:rgb(104, 104, 104);

color:white
}</string>
     </property>
     <property name="title">
      <string>Original image</string>
     </property>
     <widget class="QLabel" name="Lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>411</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:3px dashed white;
background-color:rgb(104, 104, 104);
            }</string>
      </property>
      <property name="text">
       <string>Drop an image here</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>300</y>
       <width>431</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color:rgb(104, 104, 104);

color:white;
}</string>
     </property>
     <property name="title">
      <string>Denoised Image</string>
     </property>
     <widget class="QLabel" name="debruite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>411</width>
        <height>301</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
border:3px dashed white;
background-color:rgb(104, 104, 104);
            }</string>
      </property>
      <property name="text">
       <string>Result!!</string>
      </property>
     </widget>
     <widget class="QLabel" name="psnrResult">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color:rgb(134, 134, 134);
color:white;
padding:1px
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>360</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;

background-color:rgb(104, 104, 104);
            }</string>
      </property>
      <property name="text">
       <string>PSNR:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="display">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
     </property>
     <property name="text">
      <string>Display result</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Wavelet Trasforme</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>1331</width>
       <height>601</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>35</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
background-color:rgb(104, 104, 104);
opacity:0.3;
color:white;
}</string>
     </property>
     <property name="title">
      <string>wavelet decomposition level:</string>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
      </property>
      <property name="text">
       <string>Level 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
      </property>
      <property name="text">
       <string>Level 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
      </property>
      <property name="text">
       <string>Level 1</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>generate data</string>
    </attribute>
    <widget class="QPushButton" name="generatDATA">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>15px</family>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(98, 98, 98);
    color: rgb(255, 255, 255);
    border-style: outset;
    padding: 2px;
    font: arial 15px;
    border-width: 1px;
    border-radius: 10px;
   
}
QPushButton:hover {
    background-color: white;
	color:rgb(98, 98, 98)
}</string>
     </property>
     <property name="text">
      <string>generate</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>161</y>
       <width>631</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
background:rgb(96, 96, 96);
color:white
}</string>
     </property>
     <column>
      <property name="text">
       <string>ecart-type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>beta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noise</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>seuil</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>759</x>
       <y>160</y>
       <width>551</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
